README.md

```diff
@@ -417,7 +417,7 @@ For many examples on how to use WDL see [the WDL site](https://github.com/broadi
 
 # Configuring Cromwell
 
-Cromwell's default configuration file is located at `src/main/resources/application.conf`.
+Cromwell's default configuration file is located at `core/src/main/resources/reference.conf`.
```

Comment:
This might be a phrasing thing. Our application.conf file is only config for non-cromwell stuff, e.g. akka. The reference.conf is our defaults (plus as you note copypasta) for cromwell.